doc/src/user_guide.rst

```diff
@@ -1124,7 +1181,8 @@ with
 
            *float* | *string*
 
-    ``no-slp-isot-wall`` requires
+    ``no-slp-isot-wall`` only works with ``navier-stokes`` |
+    ``ac-navier-stokes`` and requires
```

Comment:
The artificial compressibility wall BCs are called just no-slip-wall (option to specify u,v,w) and slip-wall.